core/views.py

```diff
@@ -226,6 +231,23 @@ class Query(graphene.ObjectType):
   system_health = graphene.String()
   users = graphene.List(UserObject, id=graphene.Int())
   read_and_burn = graphene.Field(PasteObject, id=graphene.Int())
+  search = graphene.List(SearchResult, keyword=graphene.String())
+
+  def resolve_search(self, info, keyword=None):
+    items = []
+    if keyword:
+      search = "%{}%".format(keyword)
```

Comment:
we can likely move this to the top level outside of the if condition and remove the other redundant search variable assignment?